CONTRIBUTING.md

```diff
@@ -52,6 +53,20 @@ Not everything is enforced or validated by the schema. A few things to pay atten
 
 The JSON files should be formatted according to the [.editorconfig](https://github.com/mdn/browser-compat-data/blob/master/.editorconfig) file.
 
+
+## Publishing a new package version
+
+Regularly, a new release of [mdn-browser-compat-data](https://www.npmjs.com/package/mdn-browser-compat-data) is created by MDN staff and will then be [deployed to the MDN site](https://github.com/mdn/browser-compat-data#browser-compatibility-tables-on-mdn). Usually this is done every Thursday (MDN never deploys to production on Fridays). Releases should be coordinated with the project owner [Florian Scholz](https://github.com/Elchi3), but anyone with merge permissions on the mdn/browser-compat-data repository has the ability to run the following steps which will create a new package version:
+
+ 1. Figure out the new version number by looking at [past releases](https://github.com/mdn/browser-compat-data/releases). The project is still sort of in alpha phase, so we're still going with patch versions for the moment. Lets assume the next version should be `0.0.43`.
+ 2. On your updated and clean master branch, run `npm version patch -m "43rd alpha version"` which creates a new patch release `0.0.43`.
```

Comment:
If I understand correctly, the later steps imply that this does a few things at once. Maybe spell it out here?

```
2. On your updated and clean master branch, run `npm version patch -m "43rd alpha version"`. Locally, this updates `package.json`, creates a new commit, and creates a new release tag.
```

Then the subsequent steps can be simplified a little ("Push the commit…", "Push the release tag…")

Reply:
I agree, this is a lot better. I've also linked to the `npm version` docs since I think it is the most important command here.